packages/wallet-sdk/src/connection/ClientMessage.ts

```diff
@@ -3,70 +3,36 @@
 
 import { IntNumber } from '../types';
 
-export interface ClientMessage {
```

Comment:
Turn on `hide whitespace`!
https://github.com/coinbase/coinbase-wallet-sdk/pull/1064/files?diff=split&w=1

Reply:
you might wanna try `Review in codespace` instead to just see the final result